### PR TITLE
Allow Rekor URL to be configured by CLI flag

### LINKED ISSUE #34: Allow the Rekor URL to be configured
Currently the `cosign.VerifyImageSignatures` call will try to use either bring-your-own PKI chain if specified via environment variable `SIGSTORE_ROOT_FILE` or use the public Fulcio chain; as instructed by the `cosign.CheckOpts` provided. A signature verifier is created with these details and used to verify the OCI signature.

As the Rekor Client in the `cosign.CheckOpts` is not nil, the code will (in the absence of a prior error), reach out to the default public Rekor to `VerifyBundle` and `tlogValidateCertificate`. The only way to use cosign-gatekeeper-provider currently requires using the public good Rekor transparency log.

Please allow the Rekor URL to be configurable so that the provider is usable in circumstances where reaching out to a public Rekor instance is not viable.

I am happy to create a PR if this suggestion is accepted.

### PR DESCRIPTION
**Pull request by richardjennings**
#### Summary
Allow the Rekor URL to be configurable #34 

#### Release Note
* Added a CLI flag `--rekor-url` to allow configuring the Rekor URL

#### Documentation

### CODE REVIEW

**Review by Dentrax — APPROVED** [review]

**Comment by richardjennings** [comment]
@Dentrax is this waiting on review by a Maintainer ?

**Comment by Dentrax** [comment]
Thanks! @evenh

**Pull request merged**
